app/javascript/packs/react-bootloader.jsx

```diff
@@ -25,6 +25,7 @@ const renderComponents = (mappings) => {
     document.querySelectorAll(selector).forEach((elem) => {
       const data = JSON.parse(elem.dataset.reactData)
       render(elem, generator(data, elem))
+      elem.classList.add('--hydrated')
```

Comment:
Do we also want to add `--hydrated` to the tooltips? If so, this call either needs to be duplicated in the `renderTooltips` function, or moved to the shared `render` function.

Reply:
Nice. Thanks. I've moved it into `render`